src/UserInterfaces/WindowsForms/WindowsDecompilerEventListener.cs

```diff
@@ -269,6 +270,35 @@ public void Error(ICodeLocation location, Exception ex, string message, params o
             diagnosticSvc.Error(location, ex, message, args);
         }
 
+        public void Error(ScriptError scriptError)
```

Comment:
Again, this code, specific to `Reko.Core.Scripts` should be in an extension method.